pkg/sdk/testint/streams_gen_integration_test.go

```diff
@@ -0,0 +1,352 @@
+		table, cleanupTable := createTable(t, client, db, schema)
+		t.Cleanup(cleanupTable)
+
+		id := sdk.NewSchemaObjectIdentifier(db.Name, schema.Name, randomAlphanumericN(t, 32))
+		req := sdk.NewCreateOnTableStreamRequest(id, table.ID())
+		err := client.Streams.CreateOnTable(ctx, req)
+		require.NoError(t, err)
+		t.Cleanup(func() {
+			err := client.Streams.Drop(ctx, sdk.NewDropStreamRequest(id))
+			require.NoError(t, err)
+		})
+
+		s, err := client.Streams.ShowByID(ctx, sdk.NewShowByIdStreamRequest(id))
+		require.NoError(t, err)
+		assert.Equal(t, "", *s.Comment)
+
+		err = client.Streams.Alter(ctx, sdk.NewAlterStreamRequest(id).WithSetComment(sdk.String("some_comment")))
+		require.NoError(t, err)
+
+		s, err = client.Streams.ShowByID(ctx, sdk.NewShowByIdStreamRequest(id))
+		require.NoError(t, err)
+		assert.Equal(t, "some_comment", *s.Comment)
+
+		err = client.Streams.Alter(ctx, sdk.NewAlterStreamRequest(id).WithUnsetComment(sdk.Bool(true)))
+		require.NoError(t, err)
+
+		s, err = client.Streams.ShowByID(ctx, sdk.NewShowByIdStreamRequest(id))
+		require.NoError(t, err)
+		assert.Equal(t, "", *s.Comment)
+
+		_, err = client.Streams.ShowByID(ctx, sdk.NewShowByIdStreamRequest(id))
+		require.NoError(t, err)
+	})
+
+	t.Run("Drop", func(t *testing.T) {
+		table, cleanupTable := createTable(t, client, db, schema)
+		t.Cleanup(cleanupTable)
+
+		id := sdk.NewSchemaObjectIdentifier(db.Name, schema.Name, randomAlphanumericN(t, 32))
+		req := sdk.NewCreateOnTableStreamRequest(id, table.ID()).WithComment(sdk.String("some comment"))
+		err := client.Streams.CreateOnTable(ctx, req)
+		require.NoError(t, err)
+
+		_, err = client.Streams.ShowByID(ctx, sdk.NewShowByIdStreamRequest(id))
+		require.NoError(t, err)
+
+		err = client.Streams.Drop(ctx, sdk.NewDropStreamRequest(id))
+		require.NoError(t, err)
+
+		_, err = client.Streams.ShowByID(ctx, sdk.NewShowByIdStreamRequest(id))
+		require.Error(t, err)
+	})
+
+	t.Run("Show terse", func(t *testing.T) {
+		table, cleanupTable := createTable(t, client, db, schema)
+		t.Cleanup(cleanupTable)
+
+		id := sdk.NewSchemaObjectIdentifier(db.Name, schema.Name, randomAlphanumericN(t, 32))
+		req := sdk.NewCreateOnTableStreamRequest(id, table.ID()).WithComment(sdk.String("some comment"))
+		err := client.Streams.CreateOnTable(ctx, req)
+		require.NoError(t, err)
+		t.Cleanup(func() {
+			err := client.Streams.Drop(ctx, sdk.NewDropStreamRequest(id))
+			require.NoError(t, err)
+		})
+
+		s, err := client.Streams.Show(ctx, sdk.NewShowStreamRequest().WithTerse(sdk.Bool(true)))
+		require.NoError(t, err)
+
+		stream, err := sdk.FindOne[sdk.Stream](s, func(stream sdk.Stream) bool { return id.Name() == stream.Name })
+		require.NoError(t, err)
+
+		assert.Equal(t, id.Name(), stream.Name)
+		assert.Equal(t, db.Name, stream.DatabaseName)
+		assert.Equal(t, schema.Name, stream.SchemaName)
+		assert.Equal(t, table.Name, *stream.TableOn)
+	})
+
+	t.Run("Show", func(t *testing.T) {
```

Comment:
it would be nice to have:
1. show for multiple
2. show for multiple with different show options (like, etc)

Reply:
Done